### ISSUE TITLE
Linux build?

### ISSUE THREAD

**Issue by Magicrafter13** [ninstar/Rich-Presence-U#6]
Any chance of this getting Linux support? GUI or not, either way.

**Comment by ninstar** [comment]
I have plans to rewrite the entire application using GTK for the GUI, so yes, at least in the future, because currently I don’t have much time.

**Comment by highjeans** [comment]
I don't think you need to rewrite it as GameMaker Studio allows building for desktop unless you don't want to pay for it.

**Comment by ninstar** [comment]
No, it wasn't for this reason, I have Linux module but GMS support for debian based distros is quite broken. They only officially support Ubuntu, and even then there are compatibility issues.

**Comment by ninstar** [comment]
It is now on Linux.
